cpp/tests/stream_compaction/stream_compaction_tests.cu

```diff
@@ -320,4 +325,209 @@ TYPED_TEST(DropNullsTest, NoNullMask)
   DropNulls<TypeParam>(
     cudf::test::column_wrapper<TypeParam>{source},
     cudf::test::column_wrapper<TypeParam> {expected});
-}
\ No newline at end of file
+}
+
+
+template <typename T>
+struct DropDuplicatesTest : GdfTest { };
```

Comment:
Make a new file `cpp/tests/stream_compaction/drop_duplicates_tests.cu` and add it to the same `STREAM_COMPACTION_TESTS` binary. The reason is that the compilation time is already getting long for `stream_compaction_tests.cu` which hurts productivity when changing the tests.

Reply:
created.